bin/example.cdn.env

```diff
@@ -14,3 +14,8 @@ CKB_NODE_URL=https://mainnet.ckb.dev
 CKB_INDEXER_URL=https://mainnet.ckb.dev/indexer
 CKB_ACP_LOCK=0xd369597ff47f29fbc0d47d2e3775370d1250b85140c670e4718af712983a2354
 CKB_EXPLORER_URL=https://explorer.nervos.org/transaction/
+
+# deposit
+COOP_TYPE_HASH=0x0cf3d57e94e5ee45aa7412e53fedaafc35656dde91803f5e1d76bf3e7d44d5d6
```

Comment:
can you add a comment to describe how the `COOP_TYPE_HASH` is calculated?
e.g.

```env
# sudt script: https://github.com/nervosnetwork/rfcs/blob/31e4679301f07ac562a57aae4ec8a1cdc6c80301/rfcs/0025-simple-udt/0025-simple-udt.md
# args: https://github.com/nervosnetwork/godwoken-info/blob/331e0c1a793489dbcb367d48523d0e9074500974/mainnet_v1/bridged-token-list.json#L515

# computeScriptHash({
#  code_hash: '...' ,
#   args: '...',
#   hash_type: '...',
# })


```

Reply:
> can you add a comment to describe how the `COOP_TYPE_HASH` is calculated? e.g.
> 
> ```shell
> # sudt script: https://github.com/nervosnetwork/rfcs/blob/31e4679301f07ac562a57aae4ec8a1cdc6c80301/rfcs/0025-simple-udt/0025-simple-udt.md
> # args: https://github.com/nervosnetwork/godwoken-info/blob/331e0c1a793489dbcb367d48523d0e9074500974/mainnet_v1/bridged-token-list.json#L515
> 
> # computeScriptHash({
> #  code_hash: '...' ,
> #   args: '...',
> #   hash_type: '...',
> # })
> ```

Ok, I will add it.